ngx-tools/type-guards/src/is-abstract-control/is-abstract-control.ts

```diff
@@ -0,0 +1,17 @@
+import { AbstractControl } from '@angular/forms';
+
+
+/**
+ * Determine if the item is an AbstractControl
+ *
+ * @param x - The item to test
+ * @return The result
+ *
+ * @example
+ * isAbstractControl(new FormControl()) // Returns: true
+ * isAbstractControl('hi')              // Returns: false
+ */
+// tslint:disable-next-line no-any
+export function isAbstractControl(x: Record<string, any>): x is AbstractControl {
```

Comment:
NOTE: Not sure why this file is reported as completely new... it did exist before.  The primary change is the type definition for `x`. It previously was `any` but TypeScript didn't like calling `hasOwnProperty` on an any (even if we coerce to verify it's not `null` or `undefined`. Changing the type to `Record<string, any>` satisfied TS. I don't _believe_ this is a breaking change as any `FormControl` or `AbstractControl` should satisfy that type.